### PR TITLE
allow disabling color with -no-color

### PR DESCRIPTION
**Pull request by nathan7**

### CODE REVIEW

**Comment by aheckmann** [comment]
+1

**Comment by Menda** [comment]
Cannot plus one, but I support this feature.

**Comment by oba11** [comment]
+1

**Comment by obitux** [comment]
+1

**Comment by ddollar** [comment]
What's the use case for this? Is there some state we could detect (non-TTY terminal?) that we could use to automatically disable color?

**Comment by oba11** [comment]
One use case is sending stdout from containers to central logging system (e.g graylog) which comes out as `[0;33;1mnginx.1 | [0m200 'OK' '/hello' 'hello' []` which makes it hard to parse e.g filtering status code, ip address, etc

**Comment by nathan7** [comment]
@ddollar forego was causing issues for Pebble's container logging (specifically, JSON logs that were parsed by another service) — I no longer have a reason to care about this PR, but I'm guessing my patched version is still in use.

**Comment by ddollar** [comment]
Closing for now because it seems out of date. Would take a PR that adjusted color output based on TTYness of the output stream.
